### PR TITLE
Refactored is_member? from User and MembershipApplication

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/152958659

Changes proposed in this pull request:  see PT story

Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

**Review by weedySeaDragon — APPROVED** [review]

**Review by thesuss — APPROVED** [review]

File: app/policies/page_policy.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def edit?
 
   private
   def user_is_member?
-    (user.is_member? || user.admin?)
+    (user.member? || user.admin?)
   end
```
Comment (RobertCram) [inline app/policies/page_policy.rb]: Maybe you could refactor this private method away. It is confusing: the method is name `user_is_member?`, but the implementation is actually the same as the already existing `user.is_member_or_admin?`

File: db/migrate/20171025191957_add_member_to_users.rb [hunk 2]
```diff
@@ -5,9 +5,13 @@ def change
...
-          user.save
+        if User.instance_methods.include?(:is_member?)
+
+          User.all.each do |user|
+            user.member = user.is_member?
+            user.save
+          end
+          
         end
       end
```
Comment (RobertCram) [inline db/migrate/20171025191957_add_member_to_users.rb]: As discussed in the team meeting, this one is no longer necessary.

File: app/policies/page_policy.rb (hunk 1 above)
Comment (patmbolger) [reply app/policies/page_policy.rb]: Good point - will do.

File: db/migrate/20171025191957_add_member_to_users.rb (hunk 2 above)
Comment (patmbolger) [reply db/migrate/20171025191957_add_member_to_users.rb]: I reverted back to prior version of this file.

**Comment by patmbolger** [comment]
@RobertCram - changes made.  Thanks for the comments.

**Review by RobertCram — APPROVED** [review]

**Pull request merged**